airflow/models/skipmixin.py

```diff
@@ -37,36 +38,23 @@
 class SkipMixin(LoggingMixin):
     """A Mixin to skip Tasks Instances"""
 
-    def _set_state_to_skipped(self, dag_run, execution_date, tasks, session):
+    def _set_state_to_skipped(self, dag_run, tasks, session):
```

Comment:
```suggestion
    def _set_state_to_skipped(self, dag_run: DagRun, tasks: Iterable[BaseOperator], session: Session):
```

This might help catch lingering usages still not passing in a DagRun… (Also need to annotate `skip` below)

Reply:
2a7f8a411